Makefile

```diff
@@ -12,7 +12,11 @@ CMD_GITHUB ?= gh
 # environment:
 ARCH ?= $(shell uname -m)
 KERN_RELEASE ?= $(shell uname -r)
-KERN_SRC ?= /lib/modules/$(KERN_RELEASE)/build
+KERN_BLD_PATH ?= $(if $(KERN_SRC),$(KERN_SRC),/lib/modules/$(KERN_RELEASE)/build)
+KERN_SRC_PATH ?= $(if $(KERN_SRC),$(KERN_SRC),/lib/modules/$(KERN_RELEASE)/source)
+ifeq ($(wildcard $(KERN_SRC_PATH)/.),)
+  KERN_SRC_PATH := $(KERN_BLD_PATH)
```

Comment:
There's a potential bug here, if the user sets KERN_SRC_PATH and it's empty, this assignment won't override it.

Reply:
Not sure I understand the bug.
KERN_SRC_PATH was already set in line 16.
We override it here only if KERN_SRC_PATH doesn't exist

Reply:
I was worried that this override won't work:

```
$ cat Makefile
FOO ?= bar
FOO := baz

yo:
	@echo $(FOO)

$ make yo FOO=hi
hi
```